app/components/FileSearchBar.tsx

```diff
@@ -0,0 +1,56 @@
+import { useState } from 'react';
+import { SearchBar } from '@rneui/themed';
+import { StyleSheet } from 'react-native';
+import { FileCardType } from "../types/FileTypes";
+import { searchFilesByKeyword } from '../utils/ServerRequests';
+
+
+type SearchBarProps = {
+  callback: () => void;
+  setSearchResultHandler: (files: FileCardType[]) => void;
+};
+
+const FileSearchBar: React.FunctionComponent<SearchBarProps> = (props: SearchBarProps) => {
+  const [search, setSearch] = useState("");
+
+  const updateSearch = async (search: string) => {
+    setSearch(search);
+    if (String(search).length < 3) {
+      return;
+    }
+    var results = await searchFilesByKeyword(search);
+
+    if (results && results !== undefined) {
+      props.setSearchResultHandler(results);
+    }
+  };
+
+  return (
+    <SearchBar
+      placeholder="Type Here..."
+      onChangeText={updateSearch}
+      value={search}
+      containerStyle={styles.searchContainer}
+      inputContainerStyle={styles.searchInput}
+      onFocus={() => props.callback()}
+      onBlur={() => setSearch("")}
+    />
+  );
+};
+
+const styles = StyleSheet.create({
+  searchContainer: {
+    height: 40,
+    padding: 0,
+    backgroundColor: "white",
+    borderColor: "#AFBBBE",
```

Comment:
Could you do me a quick favour and maybe add this color to the ``Colors`` object, maybe under the name ``lightGray``. Then also use this color for the delimiter border color in [line 170](https://github.com/lern-fair-file-sharing/app/blob/15-create-file-search-bar/app/(tabs)/files.tsx#L170)